src/command_modules/azure-cli-sf/setup.py

```diff
@@ -34,7 +34,7 @@
 DEPENDENCIES = [
     'azure-servicefabric==5.6.130',
     'azure-cli-core',
-    'adal==0.4.3'
+    'adal>=0.4.3'
```

Comment:
We don't need to fix to one version as long as the `0.4.3`  is minimum. I have PR #3183 to change this line as the current master branch is broken